requirements.txt

```diff
@@ -5,4 +5,5 @@ patsy>=0.4.0
 joblib>=0.9
 tqdm>=4.8.4
 six>=1.10.0
+h5py>=2.7.0
 enum34>=1.1.6; python_version < '3.4'
```

Comment:
yeah, I'd be surprised if anyone was actually using python 3.3 or 3.2 at this point, but figured we'd allow for that :)  More realistic would be `python_version <= 2.7`